internal/driver/driver.go

```diff
@@ -232,6 +232,21 @@ func (d *Driver) Stop(force bool) error {
 	return nil
 }
 
+func (d *Driver) AddDevice(deviceName string, protocols map[string]models.ProtocolProperties, adminState models.AdminState) error {
+	d.Logger.Warn("Driver's AddDevice function didn't implement")
```

Comment:
Please use debug level, and states
fmt.Sprintf("Device %s is added", deviceName)

Reply:
@cloudxxx8 Modified. Please help review.